### PR TITLE
Ansible workshop

### PR DESCRIPTION
**Pull request by AleksandMur**
Could you chek my home task

### CODE REVIEW

File: Aleksandr_Murygin/14.Ansible_Workshop/WS/roles/mysql/templates/database.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### {{ ansible_managed }}
```
Comment (pluhin) [inline Aleksandr_Murygin/14.Ansible_Workshop/WS/roles/mysql/templates/database.yml.j2:1]: This template belongs to application tasks, please move it to application role

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by AleksandMur** [comment]
corrected

File: Aleksandr_Murygin/14.Ansible_Workshop/WS/roles/mysql/templates/database.yml.j2 (hunk 1 above)
Comment (pluhin) [reply Aleksandr_Murygin/14.Ansible_Workshop/WS/roles/mysql/templates/database.yml.j2]: @AleksandMur

Did you test it? Please provide output

**Comment by AleksandMur** [comment]
corrected v0.2

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**